src/CurlKit/CurlDownloader.php

```diff
@@ -165,7 +165,7 @@ public function request($url, $params = array(), $options = array() )
 
         // When using HTTP TUNNEL, there is an extra response line before the 
         // original response line, we need to separate them if it matches "Connection established"
-        if (preg_match('#HTTP/1.[0-1] 200 Connection established#i', $data)) {
+        if (preg_match('#HTTP/1.[0-1] 200#i', $data)) {
```

Comment:
@c9s are you sure this is the right change? It contradicts the comment and breaks PhpBrew the build after an update to the latest version of the library. See https://github.com/phpbrew/phpbrew/pull/1038, [build #570616613](https://travis-ci.org/phpbrew/phpbrew/builds/570616613).